main.go

```diff
@@ -97,15 +105,31 @@ func main() {
 	service.Use(middleware.ErrorHandler(service, true))
 	service.Use(middleware.Recover())
 
-	publicKey, err := token.ParseRSAPublicKeyFromPEM([]byte(RSAPublicKey))
+	publicKey, err := token.ParsePublicKey(token.RSAPublicKey)
+	if err != nil {
+		panic(err)
+	}
+
+	privateKey, err := token.ParsePrivateKey(token.RSAPrivateKey)
 	if err != nil {
 		panic(err)
 	}
+
+	tokenManager := token.NewManager(publicKey, privateKey)
 	app.UseJWTMiddleware(service, jwt.New(publicKey, nil, app.NewJWTSecurity()))
```

Comment:
We should add a token validator method which would do validations on the token apart from the usual RSA decryption,  once the signature and the claims requirements are proven to be valid.

```
validationHandler, _ := goa.NewMiddleware(func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
       token := jwt.ContextJWT(ctx)
        if val, ok := token.Claims["is_uncle"].(string); !ok || val != "ben" {
            return jwt.ErrJWTError("you are not uncle ben's")
        }
    })
app.UseJWT(jwt.New("secret", validationHandler, app.NewJWTSecurity()))
```

Ref: https://github.com/goadesign/goa/blob/master/middleware/security/jwt/jwt.go

The method could be a very basic one ATM.